testserver/minecraft.js

```diff
@@ -0,0 +1,72 @@
+// Require Node.js standard library function to spawn a child process
+const spawn = require('child_process').spawn;
+let minecraftServerProcess;
+
+var isMinecraftRunning = false;
```

Comment:
You forgot a bunch of vars. All vars should either be const or let depending on whether it'll be changed or not.

Reply:
Done.